demos/TicTacToe/Classes/AppDelegate.cpp

```diff
@@ -21,6 +21,9 @@ bool AppDelegate::applicationDidFinishLaunching() {
   }
 
   auto scene = TicTacToe::createScene();
+  if (scene == NULL) {
+    exit(true);
```

Comment:
exiting the application is dangerous if it's not done in the main method as other allocations could have occurred (beyond your domian) that aren't cleaned up.  What if you return false here instead?

Also what about the code above?  Do the director and glview need to be released, destroyed, etc?